### PR TITLE
Fixed Typographical Error in guide/english/css/index.md

### PR DESCRIPTION
**Pull request by iamrajiv**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

### CODE REVIEW

File: guide/english/css/index.md [hunk 1]
```diff
@@ -30,21 +30,21 @@ and `yellow` is the style we want to give it.
 <!DOCTYPE html>
 <html>
   <head>
-	  <style>
-  		body { background-color: Yellow; }
+      <style>
```
Comment (thecodingaviator) [inline guide/english/css/index.md]: Change all the indentation to be 2 spaces per level please

**Review by thecodingaviator — CHANGES_REQUESTED** [review]

**Comment by iamrajiv** [comment]
@thecodingaviator is this PR good to be merged

**Comment by thecodingaviator** [comment]
Yup @iamrajiv

**Review by thecodingaviator — APPROVED** [review]
LGTM! 👍

**Comment by thecodingaviator** [comment]
@iamrajiv Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Pull request merged**